shared/actions/typed-routes.tsx

```diff
@@ -48,6 +48,7 @@ export const appendNewTeamBuilder = (teamID: TeamTypes.TeamID) =>
           filterServices: flags.teamsRedesign
             ? ['keybase', 'twitter', 'facebook', 'github', 'reddit', 'hackernews']
             : undefined,
+          goButtonLabel: 'Add',
```

Comment:
unrelated?

Reply:
yeah i'm wondering how that got there...